src/mon/OSDMonitor.cc

```diff
@@ -606,6 +701,18 @@ void OSDMonitor::update_from_paxos(bool *need_bootstrap)
   }
 }
 
+void OSDMonitor::register_cache_with_pcm(uint64_t target, uint64_t min)
+{
+  uint64_t max = min;
```

Comment:
The relationship here should be target > max > min. The idea is to let the cache grow up to a certain size while attempting to keep mapped memory below target. Making the max lower than target lets us account for memory fragmentation and other overhead that we can't measure directly, as in https://github.com/ceph/ceph/blob/master/src/os/bluestore/BlueStore.cc#L3658

Reply:
I have re-used the same logic as in bluestore. The config options from bluestore are re-used for calculating the max memory.